docker-monolith/infra/ansible/playbooks/reddit_microservice.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Wait 300 seconds, but only start checking after 60 seconds
+  hosts: tag_reddit-app-{{ lookup('env', 'CI_COMMIT_REF_NAME') }}
+  gather_facts: false
+  tasks:
+    - name: Wait_for_connection
+      ansible.builtin.wait_for_connection:
+        delay: 60
+        timeout: 300
+
+
+- name: Wait for infrastructure to become available
```

Comment:
Here should be "Deploy reddit microservices" (or similar to) because you are waiting infrastructure readiness above.